server/@types/shared/models/TemporaryAccommodationUserRole.ts

```diff
@@ -3,4 +3,4 @@
 /* tslint:disable */
 /* eslint-disable */
 
-export type TemporaryAccommodationUserRole = 'assessor' | 'referrer';
+export type TemporaryAccommodationUserRole = 'assessor' | 'referrer' | 'cas3_referrer';
```

Comment:
For my understanding, where's this change come from?

Reply:
removed this commit based on our discussion